packaging/linux/deb/openra.lintian-overrides

```diff
@@ -0,0 +1,3 @@
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSans.ttf also in fonts-freefont-ttf
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSansBold.ttf also in fonts-freefont-ttf
+openra: binary-without-manpage usr/games/openra-server
```

Comment:
Symlinking of fonts and JS files etc. actually happens a lot, but that is usually in a situation that upstream doesn't want to change things. As you say @Mailaender, we don't have a distribution package here, so might as well leave it like that.
I'd like to have it suppress the warning while documenting it, so if things become unclean again in the future (lintian-wise), it is noticed immediately and either fixed or documented and overridden (in very rare cases).

I am proceeding with adding some comments and then I'm quite happy with the result!